src/ring_theory/polynomial/homogeneous.lean

```diff
@@ -0,0 +1,162 @@
+/-
+Copyleft 2020 Johan Commelin. No rights reserved.
```

Comment:
In principle I like this, but I wonder whether in practice this could cause problems for users / organizations which have to follow strict rules ensuring they're not violating copyright.

Reply:
I have no idea... who should we ask to find out?

Reply:
I would ask on Zulip, I guess. I'm not too experienced with this sort of thing myself.